content/applications/general/apps_modules.rst

```diff
@@ -2,73 +2,93 @@
+app's card.
 
 .. note::
-   If the module you are looking for is not listed, you can **update the app list**.
-
-   To do so, activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Apps
-   --> Update Apps List` and click on *Update*.
+   If the app or module to be installed is not listed, update the app list by activating
+   :ref:`developer mode <developer-mode>`, and then go to :menuselection:`Apps --> Update Apps
+   List`, and then click :guilabel:`Update`.
 
 .. _general/upgrade:
 
 Upgrade apps and modules
 ========================
 
-On some occasions, new improvements or app features are added to :doc:`supported versions of Odoo
-</administration/supported_versions>`. To be able to use them, you must **upgrade** your app.
+With each :doc:`new Odoo release </administration/supported_versions>`, new improvements or app
+features are added. Upgrade the app to use these new improvements and features.
 
-Go to :menuselection:`Apps`, click on the *dropdown menu* of the app you want to upgrade, then on
-*Upgrade*.
+Go to :menuselection:`Apps` and then on the app to upgrade, click the :icon:`fa-ellipsis-v`
+:guilabel:`(vertical ellipsis)` icon and select :guilabel:`Upgrade`.
 
 .. _general/uninstall:
 
 Uninstall apps and modules
 ==========================
 
-Go to :menuselection:`Apps`, click on the *dropdown menu* of the app you want to uninstall, then on
-*Uninstall*.
+.. danger::
+   Uninstalling apps also deletes their database records. Test uninstalling apps on a duplicated
+   database before removing apps on a production database.
+
+.. note::
+   Some apps have dependencies, meaning that one app requires another. Therefore, uninstalling one
+   app may uninstall multiple apps and modules.
+
+Go to :menuselection:`Apps` and then on the app to uninstall, click the :icon:`fa-ellipsis-v`
+:guilabel:`(vertical ellipsis)` icon and select :guilabel:`Uninstall` to open the
+:guilabel:`Uninstall module` pop-up window.
+
+The :guilabel:`Apps to Uninstall` section lists the applications that will be uninstalled.
+
+.. tip::
+   Select the :guilabel:`Show All` checkbox to display all module dependencies.
+
+The :guilabel:`Documents to Delete` section lists the database records that will be deleted.
+
+To proceed with uninstalling the app, its dependencies, and all related database records, click
+:guilabel:`Uninstall`.
 
 .. image:: apps_modules/uninstall.png
-   :align: center
+   :alt: An app's card with the "Uninstall" menu option highlighted.
 
 Some apps have dependencies, meaning that one app requires another. Therefore, uninstalling one app
-may uninstall multiple apps and modules. Odoo warns you which dependent apps and modules are
-affected by it.
+may uninstall multiple apps and modules. Odoo provides a warning message showing which dependent
+apps and modules are affected by it as well as what database records get removed if the
+uninistallation is completed.
```

Comment:
You can remove this paragraph since it is covered in the note we added and the instructions above :) 

If you do keep this, please correct "uninistallation" to "uninstallation" :)